pkg/scheduler/framework/runtime/framework.go

```diff
@@ -355,6 +355,10 @@ func NewFramework(ctx context.Context, r Registry, profile *config.KubeScheduler
 		options.captureProfile(outputProfile)
 	}
 
+	m := *f.ListPlugins()
+	loggerV := logger.V(2)
+	loggerV.Info("Plugins", "config.Plugins", m)
```

Comment:
↓ is less redundant

```suggestion
	logger.V(2).Info("Plugins", "config.Plugins", *f.ListPlugins())
```

\+ also can you elaborate the log message more so that users can easily get what this log means.

Reply:
@sanposhiho Done !!